packages/firestore/src/platform/browser/random_bytes.ts

```diff
@@ -30,7 +30,7 @@ export function randomBytes(nBytes: number): Uint8Array {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     typeof self !== 'undefined' && (self.crypto || (self as any)['msCrypto']);
   const bytes = new Uint8Array(nBytes);
-  if (crypto) {
+  if (crypto && crypto.getRandomValues) {
```

Comment:
Do you mind doing this?

```suggestion
  if (crypto && typeof crypto.getRandomValues === 'function') {
```

Reply:
Sure, good idea.